app.rb

```diff
@@ -0,0 +1,147 @@
+require_relative './book'
+require_relative './teacher'
+require_relative './student'
+require_relative './person'
+require_relative './rental'
+# App object class
+class App
+  def initialize
+    @book_list = []
+    @rentals = []
+    @person = []
+  end
+
+  def menu
+    puts "Welcome to school library ! \n\n"
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts "7 - Exit\n\n"
+    gets.chomp
+  end
+
+  def menu_run
+    choice = menu.to_s
+    case choice
+    when '1'
+      list_all_books
+    when '2'
+      list_all_person
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      list_all_rentals_for_a_given_person_id
+    else
+      puts 'Thank you for using this App'
+      exit
+    end
+  end
```

Comment:
- [ ] Kindly move these functions into your `main.rb` file, this is a project requirement.
- [ ] Also kindly ensure your app is functioning am getting this error from one of your files when I run it locally
![image](https://user-images.githubusercontent.com/61408860/177016501-0c5bde73-5b1c-4650-a01a-0ea8c8c11794.png)